src/pages/en/guides/typescript.md

```diff
@@ -13,24 +13,11 @@ Astro doesn't perform any type checking itself. Type checking should be taken ca
 
 It is **strongly recommended** that you create a `tsconfig.json` file in your project, so that tools like Astro and VSCode know how to understand your project. Some features (like npm package imports) aren't fully supported in TypeScript without a `tsconfig.json` file.
 
-Some TypeScript configuration options require special attention in Astro. Below is our recommended starter `tsconfig.json` file, which you can copy-and-paste into your own project. Every [astro.new template](https://astro.new/) includes this `tsconfig.json` file by default.
+`tsconfig.json` templates are included in Astro that you can extend from using [the `extends` setting](https://www.typescriptlang.org/tsconfig#extends). They can be found here: [astro/tsconfigs/base](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/base.json), [astro/tsconfigs/strict](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/strict.json) and [astro/tsconfigs/strictest](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/strictest.json).
```

Comment:
```suggestion
Three extensible `tsconfig.json` templates are included in Astro: `base`, `strict`, and `strictest`. The `base` template powers our VSCode extension and enables several modern ESM features. We recommend using `strict` or `strictest` if you plan to write TypeScript in your project. You can view and compare the three template configurations at [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/).

To add or change a template, use [the `extends` setting](https://www.typescriptlang.org/tsconfig#extends):
```
What do you think about a structure like this? Of course, change anything as necessary to make sure it reads the way someone configuring TypeScript would expect.

Reply:
I like the structure but this sentence isn't true:

> The `base` template powers our VSCode extension and enables several modern ESM features.

What powers the VS Code extension is ultimately the template the user has chosen. As for features, the `base` template is used as a basis for the `strict` and `strictest` template (but is also usable by itself, strictest extends from strict extends from base), so every templates has the same "features" (apart from strictness related ones)

I'll take the structure here and `extends` from it, ba dum tssh

Reply:
> I'll take the structure here and `extends` from it

Careful! We're gonna start getting you to write Docs Congrats messages!  ;)

Yes, please do make the statement accurate! I noticed in the template that `strict` extends from `base`, so I assumed that the properties mostly extended each other in order (or at least, both were extended from base). So while I didn't explicitly say that `strict = base plus a strict setting`, I thought maybe that would come across, while emphasizing that the features in base gave you stuff that was useful even if you didn't write any TS in your project. You're right though, it should not sound like base and ONLY base gives you that!